<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>395</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="lineEditFood">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>30</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidgetSelectedItems">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>341</width>
     <height>221</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Ice Cream</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Apple</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Soda</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Coffee</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Chocolate</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="pushButtonAdd">
   <property name="geometry">
    <rect>
     <x>26</x>
     <y>30</y>
     <width>91</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Item</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>330</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Edit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonDelete">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>330</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonDeleteAll">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>330</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete All</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonDeleteAll</sender>
   <signal>clicked()</signal>
   <receiver>listWidgetSelectedItems</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>338</y>
    </hint>
    <hint type="destinationlabel">
     <x>300</x>
     <y>256</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
